### DISCUSSION TITLE
Colab clean Install workflow tips

### DISCUSSION THREAD

**Discussion by Maxadox** [TheLastBen/fast-stable-diffusion#2096] [Q&A]
Hi everyone, I'm looking for some workflow tips

So, after a clean install I always copy back my Lora , embeddings and textual inversion folder but I re-install manually the extensions from the web.ui, can I copy the extensions folder as well? the few times I tried it didn't seem to work, but not sure if the error was related ... how do you folks go about doing a proper and easy clean install with most of your configs ?

Thank you all so much in advanced

**Comment by Karuzzzo** [comment]
You can place your extensions in any folder on google disk, and then perform something like:

```!cp /content/gdrive/MyDrive/Loras/*  /content/gdrive/MyDrive/sd/stable-diffusion-webui/models/Lora```

With respect to your path names.
Place it somewhere inside your colab, use for all files you need)
